motoserver/Tiltfile

```diff
@@ -0,0 +1,65 @@
+load("ext://uibutton", "cmd_button")
+load("../devcore/Tiltfile", "tilt_port")
+
+extension_root = os.getcwd()
+
+
+def motoserver_up(init_script = "", labels = []):
+    """Deploy MotoServer using Docker and Kubernetes."""
+
+    docker_build(
+        "motoserver-persistence",
+        context = os.path.join(extension_root, "src/"),
+        dockerfile = os.path.join(extension_root, "Dockerfile"),
+    )
+
+    if not init_script:
+         init_script = read_file(os.path.join(extension_root, "scripts/init.sh"))
+    else:
+        # If `init_script` is a file path, return its contents. Otherwise, treat it as a string.
+        init_script = read_file(init_script, init_script)
+
+    k8s_yaml([
+        os.path.join(extension_root, "k8s/moto-statefulset.yaml"),
+        os.path.join(extension_root, "k8s/moto-service.yaml"),
+    ])
+    k8s_yaml(init_script_configmap_yaml("motoserver-init-script", init_script))
+
+    k8s_resource(
+        "motoserver",
```

Comment:
It would be nice to allow the `k8s_resource` names to be customized. That way they can be customized per app, `e.g imports-service-motoserver`

Reply:
We'll leave this for now, because it is a large undertaking and there are multiple solutions which will need to be considered.